dreamers-app/src/Components/Log.js

```diff
@@ -1,7 +1,35 @@
 import React, { Component } from 'react';
+import {Form, FormControl, FormGroup, ControlLabel, Button} from 'react-bootstrap';
+
+const logStyle = {
+  height: '20rem',
+  margin: '0 auto',
+  width: '80%',
+  paddingRight: '0.5rem'
+}
+
+const formInstance = (
```

Comment:
also this,  formInstance is very general, but this only occurs on this page and only references the dream journal form